### PR TITLE
Improvements to docs

### PR DESCRIPTION
**Pull request by fridex**
## This introduces a breaking change

- [x] No

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -63,8 +64,8 @@ Imagine you have an application that has one dependency:
   $ cat requirements.txt
   tensorflow
 
-
-Tool provided by this project will tell you how dependencies could be resolved:
```
Comment (KPostOffice) [inline README.rst:67]: This project provides a tool which tells you what dependencies...

File: README.rst [hunk 2]
```diff
@@ -74,18 +75,22 @@ The output of this solver is a dependency analysis for the given software stack
...
+specifiers and/or limit the output just to direct dependencies.
+
 Produced output
 ===============
 
-This tool (unless ``--no-transitive`` is specified) analyzes recursively all the
-dependencies of the desired project. Dependencies to be analyzed can be defined
-in similar to ``requirements.txt`` file or as a string in a form of:
+This tool (unless ``--no-transitive`` is specified) analyzes recursively all
+the dependencies of the desired package inside specific environment.
```
Comment (KPostOffice) [inline README.rst:91]: inside *a* specific environment

File: README.rst [hunk 3]
```diff
@@ -74,18 +75,22 @@ The output of this solver is a dependency analysis for the given software stack
...
+It is also possible to restrict version using standard Python version range
+specifiers and/or limit the output just to direct dependencies.
+
 Produced output
 ===============
 
-This tool (unless ``--no-transitive`` is specified) analyzes recursively all the
-dependencies of the desired project. Dependencies to be analyzed can be defined
-in similar to ``requirements.txt`` file or as a string in a form of:
+This tool (unless ``--no-transitive`` is specified) analyzes recursively all
```
Comment (KPostOffice) [inline README.rst:90]: "recursively analyzes"

**Review by KPostOffice — COMMENTED** [review]
Just a few comments

File: README.rst (hunk 3 above)
Comment (fridex) [reply README.rst]: Addressed in 65fad98 Thanks 👍🏻

File: README.rst (hunk 2 above)
Comment (fridex) [reply README.rst]: Addressed in 65fad98 Thanks 👍🏻

File: README.rst (hunk 1 above)
Comment (fridex) [reply README.rst:67]: I think this sentence was removed.

**Review by harshad16 — COMMENTED** [review]
/lgtm
:100:

**Pull request merged**